lms/djangoapps/courseware/tabs.py

```diff
@@ -52,26 +52,6 @@ def is_enabled(cls, course, user=None):
         """
         Returns true if this tab is enabled.
         """
-        if DISABLE_UNIFIED_COURSE_TAB_FLAG.is_enabled(course.id):
-            return super().is_enabled(course, user)
-        # If this is the unified course tab then it is always enabled
-        return True
-
-
-class CourseInfoTab(CourseTab):
-    """
-    The course info view.
-    """
-    type = 'course_info'
-    title = gettext_noop('Home')
-    priority = 10
-    view_name = 'info'
-    tab_id = 'info'
-    is_movable = False
-    is_default = False
-
-    @classmethod
-    def is_enabled(cls, course, user=None):
         return True
```

Comment:
CoursewareTab is_enable is always True
since DISABLE_UNIFIED_COURSE_TAB_FLAG has been removed

Reply:
If `is_enabled` cannot be False but True it becomes useless class parameter and should be removed as well

Reply:
if I remove `is_enabled` method it will be used method from `EnrolledTab`. It's the same as [`return super().is_enabled(course, user)`](https://github.com/raccoongang/edx-platform/pull/2414/files/b25c9d3c552dfa6e30c6034a777e9a6d1852c39e#diff-8be750010666701c7cff0d5615d70578f55937ce6724b65ce5415bade50f66afR26) (showing for enrolled users only). According to DISABLE_UNIFIED_COURSE_TAB_FLAG removal we should display this tab for all users.